Makefile

```diff
@@ -0,0 +1,15 @@
+all:
+	@cd $(GOPATH)/src; go install github.com/Symantec/cloud-gate/cmd/*
+
+
+format:
+	gofmt -s -w .
+
+format-imports:
+	goimports -w .
+
+
+test:
+	@find * -name '*_test.go' |\
```

Comment:
Can't we just run
```
go test ./...
```
?

Reply:
Plain `go test` is very noisy: it spams for every package without test files.

Reply:
The idea is to encourage better test coverage ;)